File: database_connection/task_list/task_list_python/database/postgres_connection.py
import psycopg2
from database.db_connection import DbConnection


class PostgresConnection(DbConnection):
  create_table_query = """
  CREATE TABLE IF NOT EXISTS task(
    id SERIAL PRIMARY KEY,
    title VARCHAR(512) NOT NULL,
    description VARCHAR(1024) NOT NULL,
    created_at TIMESTAMP
  );
  """

  def get_connection(self):
    return psycopg2.connect("dbname=task_list user=postgres password=postgres")

File: database_connection/task_list/task_list_java/src/main/java/com/database/PostgresConnection.java
package com.database;

public class PostgresConnection extends DbConnection {

  @Override
  protected void init() {
    driverClassName = "org.postgresql.Driver";
    url = "jdbc:postgresql://localhost:5432/task_list";
    user = "postgres";
    password = "postgres";
    createTableQuery = """
            CREATE TABLE IF NOT EXISTS task(
              id SERIAL PRIMARY KEY,
              title VARCHAR(512) NOT NULL,
              description VARCHAR(1024) NOT NULL,
              created_at TIMESTAMP
            );
        """;
  }
}
